### DISCUSSION TITLE
Flow type constants don't quite match engine types

### DISCUSSION THREAD

**Discussion by rowanseymour** [nyaruka/floweditor#1126] [Ideas]
Was trying to simplify engine vs database types in RP (https://github.com/nyaruka/rapidpro/commit/288e0661bdfccce4c6098ac52881f5360767b6a1) and I noticed the types the editor uses don't quite match the engine types.

Engine:

* messaging
* messaging_offline
* voice

Editor:

* message
* message_offline
* voice

Thankfully it doesn't seem to cause any problems because RP creates the flow with the engine type, and the editor doesn't change it.

**Comment by ericnewcomer** [comment]
Odd, I swear I had changed that exact bit. Maybe I changed it to be wrong!
